dotnetapp-3.5/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet-framework:3.5
+FROM microsoft/dotnet-framework:3.5 as builder
```

Comment:
This suggests to me that we need to create a 3.5 builder as well. I don't want to ship our asymmetric samples between 3.5 and 4.x (not that our samples are the decision maker!).

Reply:
I agree

Reply:
Agreed . Issue: https://github.com/Microsoft/dotnet-framework-docker-samples/issues/17 tracks the related work.